rllib/core/learner/learner.py

```diff
@@ -1093,7 +1093,7 @@ def update_from_iterator(
             )
 
         self._check_is_built()
-        # minibatch_size = minibatch_size or 32
+        minibatch_size = minibatch_size or 1
```

Comment:
I think this can be removed. We are not using this var anyways. We force-set `batch_size=1` below.

Reply:
Correct we can remove it fully.